### DISCUSSION TITLE
What are the application's privacy and data handling policies?

### DISCUSSION THREAD

**Discussion by juanchisdubdub** [upscayl/upscayl#813] [Q&A]
From the website I have not found how they are going to handle the pictures I upload to the app, if the metadata is removed or if they are shared with third parties, it would be helpful if they set them up and regulate them according to the laws where they are legally established, for greater confidence.

**Comment by aaronliu0130** [comment]
Duplicate of #782.
![compulsory rickrolls are issued for duplicates](https://user-images.githubusercontent.com/11874211/218282922-efab6795-84af-4cfb-9400-c28c8729ea78.gif)

---

Every thing is stored and processed locally on your computer. We do not send your data anywhere.

**Reply by MartenBE** [reply]
It would be nice if "_Every thing is stored and processed locally on your computer. We do not send your data anywhere._" could be placed prominently on the README. It isn't clear from the docs/website as its privacy policy also handles the cloud version and gives the impression this application also stores the images:

> 2. What We Collect
> 
> We collect the following categories of information when you use the Service:
> 
> ...
> 
> 2.4 Uploaded Content and Generated Outputs
> 
> Images you upload, links you provide for processing, and generated outputs
> File metadata (e.g., filename, MIME type, size, dimensions) and storage paths
>
> ~ https://upscayl.org/privacy
